cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
Sounds good to me. I'm not aware of any global UX heuristic that could help us here. We have to decide and be consistent regarding this: Do we want (a) arguments like "number of slices to deal with" to affect the resulting number (add so many that we have _n_ in the end)? Or (b) should they set the number for the current command (add _n_)?

(b) is simpler, more straightforward in my opinion and also a simpler promise to keep in a distributed team.